nautobot/extras/tables.py

```diff
@@ -532,6 +532,23 @@ class JobResultTable(BaseTable):
         orderable=False,
         attrs={"td": {"class": "text-nowrap report-stats"}},
     )
+    actions = tables.TemplateColumn(
+        template_code="""
+            {% load helpers %}
+            {% if record.job_kwargs %}
+                <a href="/extras/jobs/{{record.name}}/?{% url_encode record.job_kwargs.data %}" class="btn btn-xs btn-success"title="Re-run job with same arguments.">
+                    <i class="mdi mdi mdi-recycle"></i>
+                </a>
```

Comment:
Let's use the new data and URL patterns introduced in 1.3 here instead:

```suggestion
            {% if record.job_model and record.job_kwargs %}
                <a href="{% url 'extras:job_run' slug=record.job_model.slug %}?{% url_encode record.job_kwargs.data %}" class="btn btn-xs btn-success" title="Re-run job with same arguments.">
                    <i class="mdi mdi mdi-repeat"></i>
                </a>
```